libs/sdk-ui/src/base/localization/bundles/en-US.json

```diff
@@ -376,6 +376,11 @@
         "comment": "",
         "limit": 0
     },
+    "gs.header.countdown.oneDayLeft": {
```

Comment:
Sorry, I've thought again and maybe we shouldn't introduce a new translationId like this. Please delete lines 379 ~ 383.